app/controllers/bulk_upload_logs_controller.rb

```diff
@@ -0,0 +1,37 @@
+class BulkUploadLogsController < ApplicationController
+  before_action :authenticate_user!
+
+  def start
+    if in_crossover_period?
+      redirect_to bulk_upload_path(id: "year")
+    else
+      redirect_to bulk_upload_path(id: "prepare-your-file")
+    end
+  end
+
+private
+
+  def in_crossover_period?
+    FormHandler.instance.forms.values.any?(&:in_crossover_period?)
+  end
+
+  def bulk_upload_path(id:)
+    case log_type
+    when "lettings"
+      bulk_upload_lettings_log_path(id:)
```

Comment:
Could we have something like "bulk_upload_#{log_type}_log_path" instead? Wouldn't need the method. Happy to omit if it makes it less clear tho

Reply:
No worries, yeah I went back and made a few tweaks. I think your suggestion would be for the latest version:
```ruby
public_send("bulk_upload_#{log_type_for_controller(controller)}_log_path", id:)
```
I'd say its about 50/50 and you could go either way. You'd get rid of the if switch for a single line which but arguably is less easier to read. I don't think this code is high churn will remain fairly stable once implemented so don't think the implementation here matters much